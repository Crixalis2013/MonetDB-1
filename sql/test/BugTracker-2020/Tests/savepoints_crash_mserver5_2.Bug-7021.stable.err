stderr of test 'savepoints_crash_mserver5_2.Bug-7021` in directory 'sql/test/BugTracker-2020` itself:


# 12:00:17 >  
# 12:00:17 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-154330" "--port=38327"
# 12:00:17 >  


# 12:00:17 >  
# 12:00:17 >  "Done."
# 12:00:17 >  

