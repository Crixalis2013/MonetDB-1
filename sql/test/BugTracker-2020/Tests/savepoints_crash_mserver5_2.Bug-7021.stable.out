stdout of test 'savepoints_crash_mserver5_2.Bug-7021` in directory 'sql/test/BugTracker-2020` itself:


# 12:00:17 >  
# 12:00:17 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-154330" "--port=38327"
# 12:00:17 >  

#start transaction;
#create table savepointtest (id int, primary key(id));
#savepoint name1;
#insert into savepointtest values(1), (2), (3);
[ 3	]
#insert into savepointtest values(4), (5), (6);
[ 3	]
#insert into savepointtest values(7), (8), (9);
[ 3	]
#savepoint name3;
#select * from savepointtest;
% sys.savepointtest # table_name
% id # name
% int # type
% 1 # length
[ 1	]
[ 2	]
[ 3	]
[ 4	]
[ 5	]
[ 6	]
[ 7	]
[ 8	]
[ 9	]
#commit;
#select 42;
% .%2 # table_name
% %2 # name
% tinyint # type
% 2 # length
[ 42	]
#select * from savepointtest;
% sys.savepointtest # table_name
% id # name
% int # type
% 1 # length
[ 1	]
[ 2	]
[ 3	]
[ 4	]
[ 5	]
[ 6	]
[ 7	]
[ 8	]
[ 9	]

# 12:00:17 >  
# 12:00:17 >  "Done."
# 12:00:17 >  

